src/RoadRegistry.BackOffice.UI/src/DownloadProduct.elm

```diff
@@ -0,0 +1,232 @@
+module DownloadProduct exposing (Msg(..), init, main, subscriptions, update, view)
+
+import Alert exposing (AlertKind(..), AlertModel, AlertMsg(..), hideAlert, showError, viewAlert)
+import Browser
+import Bytes exposing (Bytes)
+import File.Download
+import Filesize
+import Footer
+import Header exposing (HeaderModel)
+import Html exposing (Html, a, div, h1, h2, li, main_, section, span, text, ul)
+import Html.Attributes exposing (class, classList, id, style)
+import Html.Events exposing (onClick)
+import Http
+import HttpBytes
+
+main: Program String Model Msg
+main =
+    Browser.element { init = init, update = update, view = view, subscriptions = subscriptions }
+
+
+type alias DownloadModel =
+    { title : String
+    , url : String
+    , downloading : Bool
+    , progressing : Bool
+    , progress : String
+    }
+
+
+type alias Model =
+    { header : HeaderModel
+    , download : DownloadModel
+    , alert : AlertModel
+    }
+
+
+init : String -> ( Model, Cmd Msg )
+init url =
+    ( { header = Header.init |> Header.downloadBecameActive
```

Comment:
There are now 2 download pages - think `Header.downloadBecameActive` will need to be deduped.